nibabel/nicom/tests/test_dicomwrappers.py

```diff
@@ -9,12 +9,17 @@
 
 import numpy as np
 
+have_dicom = True
 try:
-    import dicom
+    import dicom as pydicom
+    read_file = pydicom.read_file
 except ImportError:
-    have_dicom = False
-else:
-    have_dicom = True
+    try:
+        import pydicom
+    except ImportError:
+        have_dicom = False
+    else:
+        from pydicom.dicomio import read_file
```

Comment:
This is a little ugly, reflecting the need to discover two different pydicom versions. The try/except/else allows us to detect a failure to import `pydicom` separately from a failure to find `pydicom.dicomio.read_file`.